lib/google-vision-client.ts

```diff
@@ -0,0 +1,5 @@
+process.env.GOOGLE_APPLICATION_CREDENTIALS = process.env.GOOGLE_APPLICATION_CREDENTIALS || 'three-monkeys-hackweek-844be021a66d.json';
+import vision from '@google-cloud/vision';
+
+const client = new vision.ImageAnnotatorClient();
```

Comment:
I was unable to use [nock](https://github.com/nock/nock) in tests. It just straight up was unaware of the HTTP requests that were made by google's library :(

I think b/c google under the hood is not using node's `http` module.

Anyone know more about this than I do?